hazelcast/src/main/java/com/hazelcast/instance/DefaultAddressPicker.java

```diff
@@ -187,6 +170,33 @@ private AddressDefinition pickAddress(NetworkConfig networkConfig) throws Unknow
         return addressDef;
     }
 
+    private AddressDefinition pickInterfaceAddress(NetworkConfig networkConfig) throws UnknownHostException, SocketException {
```

Comment:
This method is slightly re-ordered to get rid of the CheckStyle issues. Please check if I did this correctly :)